src/TLuaInterpreter.cpp

```diff
@@ -8127,7 +8119,7 @@ int TLuaInterpreter::tempAnsiColorTrigger(lua_State* L)
         }
     }
 
-    if (lua_gettop(L) > 2 && !lua_isnumber(L, 2)) {
+    if (!lua_isnumber(L, 2)) {
```

Comment:
:bulb: You know what! I think that was why I had the code as the first argument so that by omitting the then third (background colour) argument it would be treated as the ignore option - as in a typical usage the user may well be only specifying the foreground colour to match against...

OTOH No-one has reported the errors in this function in the past so the usage must have been negligible!

Reply:
^^^ This change is still needed - and the line numbers have not changed...

Reply:
Fixing.